src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -24,7 +24,7 @@ public class AnalysisService : IDisposable
     {
         #region Private Fields
 
-        private const int NumRunspaces = 2;
+        private const int NumRunspaces = 1;
```

Comment:
Hmmm, that's interesting...  I hadn't considered that before, maybe it would be useful in a broader context in PSES.  Thanks for the tip!